### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by aron-helu**
Hello :smiley: :smiley:

Web page for the desktop version. using media queries for 768px and wider. 
To lay out the page I use grid and flexbox. There is no implementation of any functionality of javascript only HTML and CSS. 

Thank you cheers.  :thumbs-up: :thumbsup:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,168 +41,178 @@ <h2 class="support-text">Software Developer</h2>
                     coded, don't hestiate to contact me.
```
Comment (Rustamxon7) [inline index.html:41]: - [ ] GOOD JOB SO FAR 🎉👌🏼👏🏼🙌🏼💥. The header description looks moved a little to the left. To fix this you can remove `margin-right: 10%` from `.text-container` class 👍🏼 .

**Screenshot**
image

File: index.html [hunk 2]
```diff
@@ -34,168 +41,178 @@ <h2 class="support-text">Software Developer</h2>
...
+                        <ul class="icon">
+                            <li><img src="./image/css.svg" alt="css-icon"></li>
+                            <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                            <li><img src="./image/html.svg" alt="html-icon"></li>
+                            <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                            <li><img src="./image/bootstrap.svg" alt="bootstrap-icon"></li>
+                            <li><img src="./image/Vector-5.svg" alt="arrow-icon"></li>
+                            <li><img src="./image/Ruby.svg" alt="ruby-icon"></li>
+                        </ul>
+                        <button type="button" class="see-project">See Project</button>
```
Comment (Rustamxon7) [inline index.html:174]: - [ ] Please also add hover and focus styles to all the buttons as given in the Figma design. 
Buttons


- [ ] [OPTIONAL] For accessibility concerns cursor should change to pointer 👆🏼 when you move the cursor over to the button. You can fix it by adding `cursor: pointer;` to the buttons.

File: index.html [hunk 3]
```diff
@@ -10,20 +10,27 @@
...
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500&family=Roboto:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
         <link rel="stylesheet" href="./style.css" />
         <title>Portfolio Aron</title>
     </head>
     <body>
 
         <!-- Header of the page with navigation bar -->
+        <header class="top-navigation">
```
Comment (Rustamxon7) [inline index.html:20]: - [ ]  [OPTIONAL] Please don't forget to add the LICENSE file 🙌🏼

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi @aron-helu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_

emojibest_com_AnimatedSticker


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aron-helu** [comment]
I am making changes for the text container to be entered. Put focus and hover on all buttons. all which asked by the code reviewer are done.

**Review by Rustamxon7 — APPROVED** [review]
## Hi @aron-helu  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**